miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -104,23 +104,27 @@ abstract class MiniAppMessageBridge {
         }
     }
 
-    /** Handle the message from external. **/
+    @SuppressWarnings("UndocumentedPublicFunction", "LongMethod")
     @JavascriptInterface
     fun postMessage(jsonStr: String) {
         val callbackObj = Gson().fromJson(jsonStr, CallbackObj::class.java)
 
-        when (callbackObj.action) {
-            ActionType.GET_UNIQUE_ID.action -> onGetUniqueId(callbackObj)
-            ActionType.REQUEST_PERMISSION.action -> onRequestPermission(callbackObj)
-            ActionType.REQUEST_CUSTOM_PERMISSIONS.action -> onRequestCustomPermissions(jsonStr)
-            ActionType.SHARE_INFO.action -> onShareContent(callbackObj.id, jsonStr)
-            ActionType.LOAD_AD.action -> adBridgeDispatcher.onLoadAd(callbackObj.id, jsonStr)
-            ActionType.SHOW_AD.action -> adBridgeDispatcher.onShowAd(callbackObj.id, jsonStr)
-            ActionType.GET_USER_NAME.action -> userInfoBridgeDispatcher.onGetUserName(callbackObj.id)
-            ActionType.GET_PROFILE_PHOTO.action -> userInfoBridgeDispatcher.onGetProfilePhoto(callbackObj.id)
-            ActionType.GET_ACCESS_TOKEN.action ->
-                userInfoBridgeDispatcher.onGetAccessToken(callbackObj.id, miniAppInfo.id)
-            ActionType.SET_SCREEN_ORIENTATION.action -> screenBridgeDispatcher.onScreenRequest(callbackObj)
+        try {
+            when (callbackObj.action) {
+                ActionType.GET_UNIQUE_ID.action -> onGetUniqueId(callbackObj)
+                ActionType.REQUEST_PERMISSION.action -> onRequestPermission(callbackObj)
+                ActionType.REQUEST_CUSTOM_PERMISSIONS.action -> onRequestCustomPermissions(jsonStr)
+                ActionType.SHARE_INFO.action -> onShareContent(callbackObj.id, jsonStr)
+                ActionType.LOAD_AD.action -> adBridgeDispatcher.onLoadAd(callbackObj.id, jsonStr)
+                ActionType.SHOW_AD.action -> adBridgeDispatcher.onShowAd(callbackObj.id, jsonStr)
+                ActionType.GET_USER_NAME.action -> userInfoBridgeDispatcher.onGetUserName(callbackObj.id)
+                ActionType.GET_PROFILE_PHOTO.action -> userInfoBridgeDispatcher.onGetProfilePhoto(callbackObj.id)
+                ActionType.GET_ACCESS_TOKEN.action -> userInfoBridgeDispatcher.onGetAccessToken(callbackObj.id)
+                ActionType.SET_SCREEN_ORIENTATION.action -> screenBridgeDispatcher.onScreenRequest(callbackObj)
+            }
+        } catch (e: UninitializedPropertyAccessException) {
+            if (this::bridgeExecutor.isInitialized)
+                e.message?.let { bridgeExecutor.postError(callbackObj.id, it) }
```

Comment:
This catch only happens when hostapp calls postMessage function so bridgeExecutor never be initialized, we don't need those executions.

There is no use case for miniapp triggers this error but as I mentioned we have this [issue](https://github.com/rakutentech/android-miniapp/pull/186#pullrequestreview-523035441) on `userInfoBridgeDispatcher`. Please make a fix for `userInfoBridgeDispatcher`.

Reply:
> This catch only happens when hostapp calls postMessage function so bridgeExecutor never be initialized, we don't need those executions.

Ok, I have removed that try/catch block if we don't need to catch initialization error for `bridgeExecutor`. 

> There is no use case for miniapp triggers this error but as I mentioned we have this issue on userInfoBridgeDispatcher. Please make a fix for userInfoBridgeDispatcher, even though the catch already covers this error but we don't want the userInfoBridgeDispatcher is uninitialized when miniapp webview init.

Also, added an extra initialization for `userInfoBridgeDispatcher` inside `MiniAppMessageBridge.init`. Could you please review it when you'll have time? Thanks.